plugins/woo-ai/src/constants.ts

```diff
@@ -1,3 +1,3 @@
-export const MIN_TITLE_LENGTH = 20;
+export const MIN_TITLE_LENGTH = 15;
```

Comment:
The title generation still comes in below 20 characters (not sure GPT 3.5 can count) so I'm lowering this.